building.sk

```diff
@@ -752,6 +752,7 @@ caves and cliffs update part 1:
 	deepslate lapis [lazuli] ore¦s = minecraft:deepslate_lapis_ore
 	deepslate diamond ore¦s = minecraft:deepslate_diamond_ore
 	
-	[any] stone ore = any ore
+	[any] stone ore = gold ore, iron ore, coal ore, lapis ore, diamond ore, redstone ore, emerald ore, copper ore
+	[any] nether ore = nether gold ore, nether quartz ore
```

Comment:
I didn't think of this until now, but this can probably be declared in the 1.16 section - when nether gold ore was added.
After this line: https://github.com/SkriptLang/skript-aliases/blob/0c926c70e232849462efa4c8c741692e8c9c90ad/building.sk#L639

Reply:
ok